sky/cli.py

```diff
@@ -1714,12 +1714,12 @@ def cost_report(all: bool):  # pylint: disable=redefined-builtin
     means if the cluster is UP, successive calls to cost-report will show
     increasing price.
 
-    The estimated cost is calculated based on the local cache of the cluster
-    status, and may not be accurate for:
+    This CLI is experimental: The estimated cost is calculated based on the
```

Comment:
nit: Consider using`-` or `.` to separate the sentences, since `:` is also used at the end of this sentence

Reply:
Reworded.